specification/fluidrelay/resource-manager/Microsoft.FluidRelay/preview/2021-03-12-preview/fluidrelay.json

```diff
@@ -0,0 +1,718 @@
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "isDataAction": {
+          "description": "Indicates whether the operation is a data action",
+          "type": "boolean"
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result of GET request to list FluidRelay operations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of FluidRelay operations supported by the Microsoft.FluidRelay provider.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationResult"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next set of operation list results if there are any.",
+          "type": "string"
+        }
+      }
+    },
+    "FluidRelayServerUpdate": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "readOnly": false,
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        }
+      },
+      "description": "The updatable properties of a Fluid Relay server"
+    },
+    "FluidRelayServerList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FluidRelayServer"
+          },
+          "description": "A sequence of FluidRelay servers.",
+          "x-adl-name": "FluidRelayServer[]"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "A link to the next page of results, if any."
+        }
+      },
+      "description": "Paged response",
+      "required": [
+        "value"
+      ],
+      "x-adl-name": "Page<FluidRelayServer>"
+    },
+    "FluidRelayServer": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FluidRelayServerProperties",
+          "description": "The resource-specific properties for this resource.",
+          "readOnly": false,
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "System meta data for this resource, including creation and modification information.",
+          "readOnly": true
+        }
+      },
+      "description": "A FluidRelay Server.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "x-ms-azure-resource": true
+    },
+    "FluidRelayServerKeys": {
+      "type": "object",
+      "properties": {
+        "key1": {
```

Comment:
>key1 [](http://example.com/codeflow?start=9&length=4)

Tag with x-ms-secret: true.

Reply:
Talked with ADL team before, and was told "format": "password" is more universally understood and it serves the same meaning.

Reply:
Current guidance still includes x-ms-secret:true.

Wiki link: [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=operationResults#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=operationResults#common-issues-found-in-review)

Reply:
Added in the latest update. Thanks.